list_books.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
```

Comment:
Observations: The code directly uses raw SQL queries, which is not a best practice.

Recommendations:
- **Use Prepared Statements**: Utilize prepared statements for the `SELECT` query to prevent SQL injection.
- **Example**:
  ```php
  $stmt = $connection->prepare("SELECT * FROM books");
  $stmt->execute();
  $result = $stmt->get_result();
  ```